samueljsb/toy-event-sourced-settings
# use sequence indexes to manage concurrency

- Move event attributes so we don't make assumptions
- Move factories to testing package
- Use factories in services tests setup
- Use factories in projections tests
- Add an index attribute to all events
- Retrieve settings as objects
- Use previous event's index to set new event's index
- Add models to track sequence numbers
- Add a trigger to ensure indexes always increase

TODO: better tests?
